### ISSUE TITLE
Graphing for estimator from Callaway and Sant’Anna (2020)

### ISSUE THREAD

**Issue by Hanh7** [jonathandroth/pretrends#4]
Hi there,

I am working on doubly robust DID (DRDID) estimator learning from Callaway and Sant’Anna (2020) and want to graph pre-trends for my dataset using library(pretrends).
The package pretrends requires to have sigma = covariance in the function. However, the result from the DRDID (I used package DID by Callaway and Sant’Anna in R) seems not to have covariance matrix.
I read that this package can be used for this DRDID.
Could you please give me further instruction?

Thank you.

**Comment by Hanh7** [comment]
Hello, 

I have received your message. (I closed the issue there since I realized that I was asking about pretrends not the honestdid, I should put the question in the correct path).

This is my result after running DRdid. I have multiple treatment periods, in this case it is 5.
<img width="946" alt="image" src="https://github.com/jonathandroth/pretrends/assets/68261868/a55a23a7-c852-41bb-8356-bc6d1fbaaf7d">

After saving the result of each estimator, I look for "sigma" in this result, but it does not show up. I think perhaps, they don't have this matrix?! 
![image](https://github.com/jonathandroth/pretrends/assets/68261868/f0cf3e54-ff25-40d8-9220-510f0823395a)
